core/src/com/unciv/logic/map/MapParameters.kt

```diff
@@ -49,6 +57,46 @@ class MapSizeNew {
         this.radius = getEquivalentHexagonalRadius(width, height)
 
     }
+
+    /** Check custom dimensions, fix if too extreme
+     * @param worldWrap whether world wrap is on
+     * @return null if size was acceptable, otherwise untranslated reason message
+     */
+    fun fixUndesiredSizes(worldWrap: Boolean): String? {
+        if (name != Constants.custom) return null  // predefined sizes are OK
+        // world-wrap mas must always have an even width, so round down silently
+        if (worldWrap && width % 2 != 0 ) width--
+        // check for any bad condition and bail if none of them
+        val message = when {
+            worldWrap && width < 32 ->    // otherwise horizontal scrolling will show edges, empirical
+                "World wrap requires a minimum width of 32 tiles"
```

Comment:
Every message we display to the user needs to be translatable... and the more messages, the more translations :)
I think that "The provided map dimensions are too small" is an acceptable catch-all for the various 'too small' clauses

Reply:
Um - I may have been intimidated by our visitors over in the issue chat... I wanted only one of them... Will do.